packages/cadl-ranch-specs/http/models/readonly-properties/main.cadl

```diff
@@ -64,19 +64,52 @@ model OutputModel {
   optionalReadonlyIntList?: int32[];
 }
 
-@doc("Round-trip model with readonly properties.")
+@doc("Input model for setting values.")
+model InputModel {
```

Comment:
I don't fully see the point of the `InputModel`, because all it is is flattened into the `RoundTripModel`. I think it's valid to have a test for a model with only `readonly` properties, and a model that has a mix of both normal properties and readonly properties, but I don't see the point of a model with only normal properties for the `readonly-properties` test

Reply:
Hi, the purpose here is to mimic the scenario when a user puts a resource with non-readonly properties(those from `InputModel` and get a response from server with both the readonly properties and the non readonly ones(like create a resource with `name` properties and get a response containing `id`, `name`, etc).

You are right, maybe I'll remove the `InputModel` and flatten the properties to `RoundTripModel` instead.